instrumentation/opentelemetry-instrumentation-fastapi/README.rst

```diff
@@ -51,6 +51,11 @@ Usage
 
     FastAPIInstrumentor.instrument_app(app)
 
+You can also pass the list of urls to exclude explicitly to the instrumentation call:
+
+.. code-block:: python
+    from opentelemetry.util.http import ExcludeList
+    FastAPIInstrumentor.instrument_app(app, ExcludeList("client/.*/info,healthcheck"))
```

Comment:
Why not allow users to pass in a sequence of strings instead? Basically what can be passed into the env var should be replicated here.

Reply:
No real reason, I can change it to work that way.

Reply:
I think it is preferred. Also users don't have to import `ExcludeList` either.

Reply:
Done